tlatools/org.lamport.tlatools/src/tlc2/output/MP.java

```diff
@@ -529,9 +529,10 @@ private static String getMessage0(int messageClass, int messageCode, String[] pa
         	}
             break;
         case EC.TLC_INVARIANT_CONSTANT_LEVEL:
-			b.append(
-					"The invariant %1% is a constant-level formula (i.e., it contains no variables, primes, or temporal operators) and evaluates to %2%. "
-					+ "To assert constant-level formulas in your spec, use ASSUME ConstInv, or to give the assumption a name such as YourAssumption, use ASSUME YourAssumption == ConstInv.");
+		b.append(
+				"The invariant %1% is a constant-level formula (i.e., it contains no variables, primes, or temporal operators) and evaluates to %2%. "
+				+ "To assert constant-level formulas in your spec, use ASSUME ConstInv. If you optionally want to give the assumption a name, write ASSUME YourAssumption == ConstInv instead. "
+				+ "See https://explain.tlapl.us/assumptions-and-invariants for additional details.");
```

Comment:
BTW, `https://explain.tlapl.us/` gives 404. It would be nice to see the list of available articles instead.

Reply:
https://github.com/tlaplus/tlaplus/issues/1266